### PR TITLE
#93 Rerender bug fix

### PR DESCRIPTION
**Pull request by WiljamiT**

### CODE REVIEW

File: src/components/screens/forms/form-screen.tsx [hunk 1]
```diff
@@ -228,9 +228,7 @@ const FormScreen: React.FC<Props> = () => {
    */
   const setFieldValue = (fieldName: string, fieldValue: FieldValue) => {
     if (formValues[fieldName] !== fieldValue) {
-      formValues[fieldName] = fieldValue;
-      
-      setFormValues({ ...formValues });
+      setFormValues({ ...formValues, [fieldName]: fieldValue }); // aiheuttaa renderöinnin [fieldName]: fieldValue
```
Comment (EVP1989) [inline src/components/screens/forms/form-screen.tsx:231]:
```suggestion
      setFormValues({ ...formValues, [fieldName]: fieldValue });
```

**Review by EVP1989 — COMMENTED** [review]
Added comment and fix the security error

File: src/components/screens/forms/form-screen.tsx (hunk 1 above)
Comment (WiljamiT) [reply src/components/screens/forms/form-screen.tsx:231]: Done.

File: src/metaform-react/MetaformComponent.tsx [hunk 2]
```diff
@@ -52,16 +52,9 @@ const MetaformComponent: React.FC<Props> = ({
...
 }) => {
-  /**
-   * Returns unique id
-   * 
-   * @returns unique id
-   */
-  const getUniqueId = () => {
-    return Math.random().toString(36).substr(2);
-  };
+  const metaformId = form.id ? `${form.id}-` : "";
```
Comment (jarinykanen) [inline src/metaform-react/MetaformComponent.tsx:55]:
```suggestion
  const metaformId = form.id ? `${form.id}` : "";
```

**Review by jarinykanen — APPROVED** [review]

**Pull request merged**